libsolidity/analysis/ConstantEvaluator.cpp

```diff
@@ -269,6 +269,20 @@ std::optional<TypedRational> ConstantEvaluator::evaluate(
 	return ConstantEvaluator{_errorReporter}.evaluate(_expr);
 }
 
+std::optional<TypedRational> ConstantEvaluator::tryEvaluate(Expression const& _expr)
+{
+	ErrorList l;
+	ErrorReporter e(l);
```

Comment:
I see it was copied from `SMTEncoder.cpp`, but if you're touching it, I'd apply our coding style and remove abbreviations:

```suggestion
	ErrorList errorList;
	ErrorReporter errorReporter(errorList);
```

Reply:
Similarly `t` in `isConstant()` would be much better off named something like `typedRational` since type is obscured by `auto`.

Reply:
Abbreviations have been expanded.